### DISCUSSION TITLE
TJpgDec.setCallback() return value

### DISCUSSION THREAD

**Discussion by picclock** [Bodmer/TFT_eSPI#1573] [Q&A - General] [answered]
Hi Bodmer
Many thx for all your hard work, and a very useful library.
I need to be able to test the touch screen during jpeg image display time.
The TJpeg.setcallback calls a function (from example tft_output) which pushes a block of pixels to the screen, but also returns a value of 1. 
Is setting the return value to 0 (zero) the correct way of aborting the image upload ? 
Any help much appreciated.
Best Regards

**Comment by Bodmer** [comment, marked as answer]
Yes, example of use is here:
https://github.com/Bodmer/TJpg_Decoder/blob/master/examples/LittleFS/LittleFS_Jpg/LittleFS_Jpg.ino#L30-L31

**Reply by picclock** [reply]
Hi Bodmer
Many Thanks.
I don't know how I missed that. 
Best Regards
picclock
